packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1545,3 +1545,31 @@ async fn unit_type_enums() {
         .unwrap();
     assert_eq!(result.value, BimBamBoum::Boum());
 }
+
+#[tokio::test]
+// TODO(vnepveu): test for multiple assets
```

Comment:
nit: I'd prefer creating an issue for it, and linking the TODO comment to it

(TODOs tend to get ignored :) )